docs/conf.py

```diff
@@ -69,12 +69,15 @@
 
 # following avoids an error where documentation is having trouble resolving
 # inherited members (methods/properties) from Celery Task in 'cowbird.request_task.RequestTask'
+
+
```

Comment:
You can move it within `autodoc_skip_member` to ensure they stay together. 
We must avoid having a new function or variable inserted in between that will become confusing.